web/sites/default/settings.php

```diff
@@ -886,6 +886,9 @@
 #   include $app_root . '/' . $site_path . '/settings.local.php';
 # }
 
+if (FALSE !== getenv('GSA_AUTH_CLIENT_SECRET')) {
```

Comment:
@casey-rapnicki-bixal Here's what I used for the name of the environment variable if you want to set the Github variable.